examples/Servo-Stop/Servo-Stop.ino

```diff
@@ -0,0 +1,36 @@
+#include <Servo.h>
+#define trigPin  4
+#define echoPin  2
+
```

Comment:
```suggestion
byte servoPin = 9;

```
Since you are defining constants for the ultrasonic sensor's pins, it makes sense to do the same for the servo for the sake of consistency.

Reply:
PR Updated